index.js

```diff
@@ -3,10 +3,21 @@ const path = require('path');
 const electron = require('electron');
 const unusedFilename = require('unused-filename');
 const pupa = require('pupa');
+const extList = require('ext-list');
 
 const app = electron.app;
 const shell = electron.shell;
 
+function getFilenameFromMime(name, mime) {
+	const ext = Object.keys(extList()).filter(x => extList()[x] === mime);
+
+	if (ext.length !== 1) {
+		return name;
+	}
```

Comment:
Would prefer this code not being here. Maybe add something like https://github.com/jshttp/mime-types#mimeextensiontype to `ext-list`?

Maybe also `ext-list` could depend on https://github.com/jshttp/mime-db, so you didn't have to keep it up to date?